util.py

```diff
@@ -106,6 +106,47 @@ def _cli():
     return bool(ctx().args and hasattr(ctx().args, '._cli') and ctx().args._cli)
 # pylint: enable=no-member
 
+KNOWN_FORMATS = {
+    'bold red': '\033[91m\033[1m',
+    'bold yellow': '\033[33m\033[1m',
+}
+
+# A collection of string-identifiers of terminals capable of rendering the defined formats
+FORMATTING_COMPATIBLE_TERMINALS = ['xterm']
```

Comment:
use a nested dict to express to which terminal type the format-sequences belong
also: consider using enums